### DISCUSSION TITLE
Importing RMDU help

### DISCUSSION THREAD

**Discussion by aSempruch** [bengtmartensson/IrScrutinizer#458] [Q&A] [answered]
Hello,

The README mentions that RMDU file import is (partially) supported, however I can't seem to figure out what menu option will allow me to import an RMDU file. Any help would be appreciated.

I also found the codes I need using the Import -> Control Tower Database option. I was able to "Load" the codes and they appear in the file browser under the import tab, however all the "Import" buttons to the right are greyed out. Doesn't matter if I select one or multiple commands, they remain grey. If someone could enlighten me why that's the case I'd be grateful.

I'm using latest release [2.3.1](https://github.com/bengtmartensson/IrScrutinizer/releases/tag/Version-2.3.1)
Thanks

**Comment by bengtmartensson** [comment, marked as answer]
> The README mentions that RMDU file import is (partially) supported, however I can't seem to figure out what menu option will allow me to import an RMDU file.

The rmdu import has been removed since the current RMIR/RMDU can export Girr files directly, which can be imported in IrScrutinizer. This is a much better solution. ~Thanks for reminding me about updating the documentation.~ The documentation has been updated, but it is not yet checked in.

> ... using the Import -> Control Tower Database option. I was able to "Load" the codes and they appear in the file browser under the import tab, however all the "Import" buttons to the right are greyed out. 

You have to give more details than that; I cannot reproduce.

**Reply by aSempruch** [reply]
Thank you for clarifying. The Girr export/import works perfectly!

As for the other issue I believe it's due to Global Cache requiring an account and only sending codes over email so IrScrutinizer can't download them directly. I just used the Global Cache website directly to get the codes.

Thanks!
